xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2647,6 +2648,37 @@ absl::Status IrEmitterUnnested::EmitHloInstruction(
               streams.destination_stream_id, streams.source_stream_id));
           return EmitFusion(Cast<HloFusionInstruction>(wrapped));
         }
+        case HloOpcode::kCall: {
+          const ExecutionStreamAssignment& stream_assignment =
+              ir_emitter_context_->execution_stream_assignment();
+          TF_ASSIGN_OR_RETURN(
+              auto stream, stream_assignment.GetSyncExecutionStreamId(wrapped));
+          auto ir_emitter = IrEmitterUnnested::Create(ir_emitter_context_);
+          DCHECK_EQ(wrapped->called_computations().size(), 1);
```

Comment:
Unless the check is expensive, XLA prefers using `CHECK`s to `DCHECK`s! Please change it!

Reply:
Why not TF_RET_CHECK?

Reply:
Fixed.